### DISCUSSION TITLE
Subpatch Output Port of Type Texture is not recognized correctly

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#755] [! Welcome - readme first!]
![image](https://github.com/cables-gl/cables_docs/assets/1138673/23ccf58b-5322-4472-94eb-f08afec88f4a)
i keep getting an error message that this is not a texture type while it is...

**Comment by pandrr** [comment]
<img width="946" alt="Screenshot 2024-05-15 at 12 40 57" src="https://github.com/cables-gl/cables_docs/assets/98792/4fb7fe1e-6c6c-48d3-aff9-b39da310be38">

you can select the object type in the dropdown. the new update should do this automatically when creating ports by dragging cables

**Reply by wirmachenbunt** [reply]
thanks, makes sense
